test/x86/avx2.c

```diff
@@ -9312,6 +9312,108 @@ test_simde_mm256_sllv_epi64(SIMDE_MUNIT_TEST_ARGS) {
   return 0;
 }
 
+static int
+test_simde_mm256_sra_epi16 (SIMDE_MUNIT_TEST_ARGS) {
+  static const struct {
+    const int16_t a[16];
+    const int16_t count[8];
```

Comment:
count is actually 64x2.  They basically just shove a count into the lower half of the vector.

Since values > 15 all do the same thing, even though it's a 64-bit value you really only want to use the 4 least significant bits, so to generate it I would do something like

```c
simde__m128i count = simde_mm_srli_epi64(simde_test_x86_random_i64x2(), 60);
```